mimic/index-dot-html.go

```diff
@@ -0,0 +1,966 @@
+// File generated by 2goarray v0.1.0 (http://github.com/cratonica/2goarray)
+
+package mimic
+
+var indexDotHTML []byte = []byte{
```

Comment:
Extra file brings lots of extra work. Checking its existence and integrity , copying them everywhere, etc. Embedding everything is also what Go does and it‘s a very good feature.